content/v1.20/concepts/image-configs.md

```diff
@@ -10,6 +10,35 @@ description: "Image Configs is an API for centralized control of the configurati
 Crossplane package images. It allows you to configure package manager behavior
 for images globally, without needing to be referenced by other objects.
 
+## Matching image references
```

Comment:
This feature is also in `master` builds, so the same changes should be copied to `/content/master` as well. This docs repo does not use git branches for the different versions.

Reply:
Done!